src/shared/infra/email/models/email-model.ts

```diff
@@ -0,0 +1,6 @@
+export type EmailModel = {
+  [key: string]: unknown;
```

Comment:
tipar um objeto assim funciona mas eu nao sou muito fã de fazer desse jeito, assim tem o mesmo efeito e menos verbosidade
```suggestion
  Record<string, unknown>;
```